packages/xo-server/src/api/network.js

```diff
@@ -85,34 +85,40 @@ createBonded.description =
 // ===================================================================
 
 export async function set({
-  network,
-
-  name_description: nameDescription,
-  name_label: nameLabel,
+  automatic,
   defaultIsLocked,
   id,
+  name_description: nameDescription,
+  name_label: nameLabel,
+  network,
 }) {
-  await this.getXapi(network).setNetworkProperties(network._xapiId, {
+  await this.getXapi(network)._editNetwork(network._xapiId, {
+    automatic,
+    defaultIsLocked,
     nameDescription,
     nameLabel,
-    defaultIsLocked,
+    otherConfig: network.other_config,
```

Comment:
Why?

Reply:
Because we need `otherConfig` [here](https://github.com/vatesfr/xen-orchestra/pull/3958/files#diff-46379bf00feb28fb73cd11988d8361daR35)
We have chosen to put `automatic` directly in network for ease of use, bu in xapi, we have to put in `other_config`.

Reply:
Why do we reset this field to the current value?